rpc_component/cli.py

```diff
@@ -425,7 +441,28 @@ def parse_args(args):
     )
 
     dl_parser = dep_subparsers.add_parser("download-requirements")
-    dl_parser.add_argument("--download-dir", default="./")
+    dl_parser.add_argument(
+        "--download-dir",
+        default="./",
+        help="The dependency component repo is cloned to this path (default=./)"
+    )
+
+    dt_parser = subparsers.add_parser("dependents")
+    dt_parser.add_argument(
+        "--component-name",
+        required=True,
+        help="The component name"
+    )
+    dt_parser.add_argument(
+        "--download-dir",
+        default="./",
+        help="Path to clone component repos while checking for dependents (default=./)"
+    )
+    dt_parser.add_argument(
+        "--text",
```

Comment:
I added the `--text` flag only because Matt and I were discussing the output of the tool in relation to how the text is consumed by common.groovy, and using YAML required either parsing the output via "awk / sed" (which works fine, but Matt and I thought it looked kind of hack'ish) or slurping the YAML text (which seemed overly complicated to consume a simple list).  So I thought about just not utilizing the YAML output in favor of a simple list of component names, but I didn't know if the YAML output (which also contains the url) would be preferred for other uses.  Hence, I added the `--text` option because it seemed simple and less "hack'ish".  I don't have a preference of which way to do it.  Let me know what you prefer.

Reply:
There are already a number of instance of `readYaml` (is that what you mean by slurping?), I can't see why the same method wouldn't work for this change.

Reply:
sounds good.  I just liked the word slurping...   actually I think I was subconsciously thinking about JSON slurper.